src/coreclr/tools/aot/ILCompiler.Trimming.Tests/TestCasesRunner/TrimmingArgumentBuilder.cs

```diff
@@ -48,6 +48,7 @@ public TrimmingArgumentBuilder (TestCaseMetadataProvider metadataProvider)
 			Options.FeatureSwitches.Add ("System.Text.Encoding.EnableUnsafeUTF7Encoding", false);
 			Options.FeatureSwitches.Add ("System.Diagnostics.Tracing.EventSource.IsSupported", false);
 			Options.FeatureSwitches.Add ("System.Globalization.Invariant", true);
+			Options.FeatureSwitches.Add ("System.TimeZoneInfo.Invariant", true);
```

Comment:
This is not necessary, we only put here whatever switches are enabled by default in an empty console app.

Reply:
is `System.Globalization.Invariant` wrong here ?

Reply:
System.Globalization.Invariant is fine - `dotnet new console --aot` enables invariant globalization by default and this mirrors that default.